src/patternfly/components/Page/page.scss

```diff
@@ -285,6 +295,40 @@
       display: block;
       visibility: visible;
     }
+
+    > .pf-c-button {
+      width: var(--pf-c-page__header-tools-group--m-icons--c-button--Width);
+      height: var(--pf-c-page__header-tools-group--m-icons--c-button--Height);
+      padding: unset;
+      background-color: var(--pf-c-page__header-tools-group--m-icons--c-button--BackgroundColor);
+      border-radius: var(--pf-c-page__header-tools-group--m-icons--c-button--BorderRadius);
+
+      > i {
+        vertical-align: #{pf-size-prem(-.75px)};
+      }
+
+      &:active,
+      &.pf-m-active {
+        --pf-c-page__header-tools-group--m-icons--c-button--BackgroundColor: var(--pf-c-page__header-tools-group--m-icons--c-button--active--BackgroundColor);
+      }
+
+      &:focus,
+      &.pf-m-focus {
+        --pf-c-page__header-tools-group--m-icons--c-button--BackgroundColor: var(--pf-c-page__header-tools-group--m-icons--c-button--focus--BackgroundColor);
+
+        outline-offset: var(--pf-c-page__header-tools-group--m-icons--c-button--focus--OutlineOffset);
+
+        /* stylelint-disable */
```

Comment:
this block doesn't appear to be necessary since the border color is `var(--pf-global--BackgroundColor--100)` which changes to black in a dark theme.

Reply:
I think its necessary for the focus style though, because the black border doesn't work on the grey background, see image here:
<img width="213" alt="Screen Shot 2020-01-07 at 2 36 51 PM" src="https://user-images.githubusercontent.com/20118816/71923554-3fc6ce80-315b-11ea-81c2-488923a50b18.png">